src/Symfony/Component/Finder/Expression/Regex.php

```diff
@@ -61,7 +61,7 @@ class Regex implements ValueInterface
      */
     public static function create($expr)
     {
-        if (preg_match('/^(.{3,}?)([imsxuADU]*)$/', $expr, $m)) {
+        if (preg_match('/^(.{2,}?)([imsxuADU]*)$/', $expr, $m)) {
```

Comment:
the regex class rejects empty expressions `//`, but it should not, they are perfectly valid